### DISCUSSION TITLE
support for non-apio programmers

### DISCUSSION THREAD

**Discussion by ertgtct** [FPGAwars/icestudio#823] [Ideas]
I am using icestudio and i am a big fan so far. One thing i'd like as a feature is the ability to change the programmer to another tool (executable). I have a board on which the vanilla apio-prog is not working, so it would be a better alternative than exporting the bitstream and programming manually (espacially for boards without an standart programmer, like the UPduino v1).

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.
